pixi/backend/api.js

```diff
@@ -15,17 +15,78 @@
  */
 
 const express = require('express');
+const {lint, LintMode, StatusNumber} = require('@ampproject/toolbox-linter');
+const cheerio = require('cheerio');
 const log = require('@lib/utils/log')('Pixi API');
+const LimitedRemoteFetch = require('@lib/utils/limitedRemoteFetch');
+
+const remoteFetch = new LimitedRemoteFetch({
+  requestHeaders: {
+    'Referer': 'https://amp.dev/page-experience/',
+  },
+});
+
+const execLint = async (url) => {
+  const res = await remoteFetch.fetchHtmlResponse(url);
```

Comment:
Why not use fetchDocument here?

Reply:
We pass the req.headers to the linter context.
Even if the checks we do might currently not need them I would still do this, because we have the headers at hand and the linter api supports them.

Reply:
good point!